ibc.ts/chainA/chainA.schem.json

```diff
@@ -6,7 +6,7 @@
         "validators": [
           "0xc7db8f558a87a45d3b4d2e7422abb48a27e6eeee00a0f884ca244d1d783d6ead2ca445e9ac8f08b311d78b4b4b69d48a9daf0c98d2a19e87e482f8060138ba88"
         ],
-        "timeoutPropose": 3000,
+        "timeoutPropose": 10000,
```

Comment:
Relayer can't handle one block per 3 sec?

Reply:
Relayer can handle it. I wrote the reason in the commit message. Wasn't it enough?
If you thought that some information was missing in the commit message, please tell me the point.

Below is the commit message I copied from the commit.
> Relayer should run harder to update light clients if the chain creates
block too fast.

Reply:
"Should run harder" sounds like that it can't handle in some cases. If it *can* handle but just harder, what's the problem?

Reply:
Thanks for your comment. Relayer will use more time to update the light client. I'll fix the commit's description.

Reply:
You mean Relayer becomes the bottleneck?

Reply:
I don't understand what you mean bottleneck. Let's have a call.